mlflow/protos/databricks_artifacts.proto

```diff
@@ -23,7 +23,7 @@ service DatabricksMlflowArtifactsService {
   rpc getCredentialsForRead (GetCredentialsForRead) returns (GetCredentialsForRead.Response) {
```

Comment:
Update client proto definitions to use batch-supported POST endpoints exposed by the backend.